packages/react/src/components/accordion/accordion.tsx

```diff
@@ -108,7 +110,10 @@ export const Accordion: React.FC<AccordionProps> = ({
                         expanded={expandedItemIds.includes(uniqueId)}
                         disabled={item.disabled}
                         buttonRef={buttonRefs[index]}
-                        onToggle={() => handleToggle(uniqueId)}
+                        onToggle={() => {
+                            onToggle(uniqueId);
+                            handleToggle?.(uniqueId, !expandedItemIds.includes(uniqueId));
+                        }}
```

Comment:
Ici j'appelerais seulement `handleToggle`, et `handleToggle` devrait appeler la prop `onToggle`.
```suggestion
                        onToggle={() => {
                            handleToggle?.(uniqueId);
                        }}
```